app/src/main/java/org/acmvit/gitpositive/MainViewModel.kt

```diff
@@ -0,0 +1,50 @@
+package org.acmvit.gitpositive
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+/**
+ * Created by Rooparsh Kalia on 02/10/21
+ */
```

Comment:
Please remove authorship as this is an OpenSource code, for the community, by the community 😃